### PR TITLE
fix(tauri): app load

### PR DESCRIPTION
**Pull request by endrl**
Document and fix app load.

### CODE REVIEW

**Review by ferferga — CHANGES_REQUESTED** [review]
* The icons should not be committed, since they're generated at build time. Comitting them adds unnecessary size to the repo. Add them to gitignore, which is some

* I've been thinking in moving all the building instructions into the wiki, given we're expanding the ways of using the client. The readme is already long enough and I think we should show first and foremost what it's needed for end users (the ones who build are a minority).

I know that Tauri doesn't boot right now, is that fixed here? What do you mean by "fix build"? Because it already builds without issues (simply check the checks of this [PR](https://github.com/jellyfin/jellyfin-vue/pull/2048))

File: README.md
Comment (ferferga) [inline README.md:156]: Why?

File: tauri/tauri.conf.json
Comment (endrl) [inline tauri/tauri.conf.json:23]: This section is required for me on win 10. It refused to `tauri build` , `tauri dev` worked

File: tauri/tauri.conf.json
Comment (endrl) [inline tauri/tauri.conf.json:55]: The `blob:` loaded worker refuses to load on edge webview in production. Still looking for a solution

File: README.md
Comment (endrl) [reply README.md:156]: It's indeed a keyword and works as is

File: tauri/tauri.conf.json
Comment (ferferga) [reply tauri/tauri.conf.json:23]: @endrl Check our [CI pipelines](https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/quality_checks.yml#L121) for windows, as I said before you need to generate icons first

Comment (endrl) [reply tauri/tauri.conf.json:23]: Yes, but no. Thats my output. How can CI pass this tauri check. If i delete `icons` folder `tauri dev` stops working too. No idea why ci is green
```
   Compiling jellyfin-vue-tauri v0.0.0 (C:\Users\usr\jellyfinvue\jellyfin-vue\tauri)
    Finished release [optimized] target(s) in 20.66s
thread '<unnamed>' panicked at 'the bundle config must have a `.ico` icon', src\interface\rust.rs:977:8
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Comment (ferferga) [reply tauri/tauri.conf.json:23]: @endrl But it's that after `npm run tauri:icon -w tauri`?

Comment (endrl) [reply tauri/tauri.conf.json:23]: yes, icons are build. I deleted them now and run again `tauri build`
```´
 Compiling jellyfin-vue-tauri v0.0.0 (C:\Users\usr\jellyfinvue\jellyfin-vue\tauri)
error: proc macro panicked
 --> src\main.rs:8:10
  |
8 |     .run(tauri::generate_context!())
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: failed to read icon C:\Users\usr\jellyfinvue\jellyfin-vue\tauri\icons/icon.png:
```

The build context message is clear. It makes no sense that ci works.

Comment (ferferga) [reply tauri/tauri.conf.json:23]: @endrl But don't delete them!
What we don't want is to have them commited to source (in the same manner as we have node_modules ignored) but ofc you need to have them in the local filesystem!

**Comment by endrl** [comment]
> I know that Tauri doesn't boot right now, is that fixed here?

At least for windows i can say yes (the CSP was probably a cross platform issue). The mixed content issue: tauri serves with https but jellyfin server running with http is still a violation.

File: tauri/tauri.conf.json
Comment (endrl) [reply tauri/tauri.conf.json:55]: I am fine disabling CSP for the moment. But when we publish official builds this should be revisited. Can be added to the Tasks list?

**Comment by ferferga** [comment]
@endrl What's blocking this PR for review?
Also, would you mind making the documentation part for building tauri in the wiki? I just allowed editing to everyone.

**Comment by endrl** [comment]
@ferferga i moved all dev docs to the wiki. Feel free to finalize the format. I think i blocked because of the ReadMe question.

**Comment by ferferga** [comment]
@endrl What's the state of this PR after the merged ``v1.5.0``?

**Comment by endrl** [comment]
Browser still rejects the blob (worker) because of csp. Now it loads. So you can merge it.
Tauri 1.5 just fixed the mixed content error (tauri served with https can't connect to http jellyfin-server)

**Review by ferferga — APPROVED** [review]
@endrl Thank you very much mate!

**Pull request merged**
